packages/core/scripts/sentry_utils.rb

```diff
@@ -36,3 +36,7 @@ def is_profiling_supported(rn_version)
 def should_use_folly_flags(rn_version)
   return (rn_version[:major] == 0 && rn_version[:minor] < 80)
 end
+
+def is_new_hermes_runtime(rn_version)
+  return (rn_version[:major] >= 1 || (rn_version[:major] == 0 && rn_version[:minor] >= 81))
```

Comment:
l: we may simplify this
```suggestion
  # RN 0.81+ uses the new Hermes Runtime access pattern
  return (rn_version[:major] >= 1 || rn_version[:minor] >= 81)
```

Reply:
> I copied this condition from the profiling condition on top of it

You are right. Let's keep it as is for consistency. It is also easier to read 👍